docs/src/main/sphinx/connector/hudi.rst

```diff
@@ -17,6 +17,8 @@ To use the Hudi connector, you need:
 * Network access from the Trino coordinator and workers to the Hudi storage.
 * Access to the Hive metastore service (HMS).
 * Network access from the Trino coordinator to the HMS.
+* Data files must be in the
```

Comment:
Spell out which specific ones are supported and link directly (ORC and Parquet only)